docs/pages/server-access/troubleshooting-server.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Troubleshooting Server Access
+description: Describes common issues and solutions for access to servers.
+---
+
+This section describes common issues that you might encounter in managing access to servers
+with Teleport and how to work around or resolve them.
+
+## Starting SSH sessions fails
+
+By default, if you use SSH to connect to a server, Teleport runs as the user login that starts 
+the SSH session. If starting the session fails, it might be due to the Teleport file permission
```

Comment:
I'm not sure I understand "Teleport file permission settings." Are these the file permissions in the `/var/lib/teleport` data directory? Maybe add an example after this sentence?

Reply:
File system permissions on the service, I think.